### PR TITLE
Show unaffected users for batch quota action

### PR DESCRIPTION
**Pull request by AlexAndBear**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/8532

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by JanAckermann** [comment]
Super open for code quality improvement tips

File: packages/web-pkg/src/components/Spaces/QuotaModal.vue [hunk 1]
```diff
@@ -88,6 +102,33 @@ export default defineComponent({
       }
       return $gettext('Change quota')
     })
+
+    const modalWarningMessage = computed(() => {
+      if (props.resourceType === 'user') {
+        if (props.spaces.length !== props.selectedResources.length) {
```
Comment (JanAckermann) [inline packages/web-pkg/src/components/Spaces/QuotaModal.vue:108]: 2 if conditions to open up the code for adjustments

File: packages/web-pkg/src/components/Spaces/QuotaModal.vue [hunk 2]
```diff
@@ -88,6 +102,33 @@ export default defineComponent({
...
+          return $gettext('Quota will only be applied to users who logged in at least once.')
+        }
+      }
+
+      return ''
+    })
+
+    const modalWarningContextualHelperData = computed(() => {
+      if (props.resourceType === 'user') {
+        if (props.spaces.length !== props.selectedResources.length) {
```
Comment (JanAckermann) [inline packages/web-pkg/src/components/Spaces/QuotaModal.vue:118]: ^

File: packages/web-app-admin-settings/src/components/Users/ContextActions.vue [hunk 3]
```diff
@@ -6,6 +6,9 @@
       :cancel="closeQuotaModal"
       :spaces="selectedPersonalDrives"
       :max-quota="maxQuota"
+      :selected-resources="items"
```
Comment (JammingBen) [inline packages/web-app-admin-settings/src/components/Users/ContextActions.vue:9]: This can be removed, or am I missing something? 🤔

Comment (JanAckermann) [reply packages/web-app-admin-settings/src/components/Users/ContextActions.vue:9]: Correct, dev leftovers

File: packages/web-app-admin-settings/src/views/Users.vue [hunk 4]
```diff
@@ -133,7 +133,10 @@
       v-if="quotaModalIsOpen"
       :cancel="closeQuotaModal"
       :spaces="selectedPersonalDrives"
+      :selected-resources="selectedUsers"
```
Comment (JammingBen) [inline packages/web-app-admin-settings/src/views/Users.vue:136]: Here as well 🙈 

```suggestion
```

**Review by JammingBen — APPROVED** [review]

File: packages/web-app-admin-settings/src/components/Users/ContextActions.vue [hunk 5]
```diff
@@ -66,6 +61,25 @@ export default defineComponent({
       { deep: true, immediate: true }
     )
 
+    const usersWithoutDrive = computed(() => {
+      return props.items.filter(({ drive }) => !isPersonalSpaceResource(drive as SpaceResource))
```
Comment (kulmann) [inline packages/web-app-admin-settings/src/components/Users/ContextActions.vue:65]: Could you please sort them by displayName?

**Review by kulmann — CHANGES_REQUESTED** [review]

File: packages/web-app-admin-settings/src/components/Users/ContextActions.vue (hunk 5 above)
Comment (JanAckermann) [reply packages/web-app-admin-settings/src/components/Users/ContextActions.vue:65]: wow missed that

**Review by kulmann — APPROVED** [review]
LGTM 💪

**Pull request merged**